CHANGELOG-3.0.md

```diff
@@ -43,6 +43,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Remove LegacyESVersion.V_7_10_ Constants ([#5018](https://github.com/opensearch-project/OpenSearch/pull/5018))
 - Remove Version.V_1_ Constants ([#5021](https://github.com/opensearch-project/OpenSearch/pull/5021))
 - Remove custom Map, List and Set collection classes ([#6871](https://github.com/opensearch-project/OpenSearch/pull/6871))
+- Remove mmap.extensions setting in favor of nio.extensions ([#9392](https://github.com/opensearch-project/OpenSearch/pull/9392))
```

Comment:
Please:

```suggestion
- Remove `index.store.hybrid.mmap.extensions` setting in favor of `index.store.hybrid.nio.extensions` ([#9392](https://github.com/opensearch-project/OpenSearch/pull/9392))
```

Reply:
Thanks

Reply:
Thanks @reta, more verbose and precise, the better!